build.proj

```diff
@@ -190,214 +123,123 @@
-
-    <Exec ContinueOnError="false"
-      Command="$(PowerShellCoreCommandPrefix) &quot;. $(LibraryToolsFolder)\NewOutputTypeIndex.ps1 -OutputFile $(PackageDirectory)\outputtypes.json -BuildConfig $(Configuration) &quot;" 
-      Condition="'$(CodeSign)' == 'true'"/>
-
-    <CallTarget targets="DependencyAnalysisNetcore" ContinueOnError="ErrorAndContinue" Condition="'$(RunStaticAnalysis)' == 'true'" />
-    <Exec Command="$(PowerShellCommandPrefix) &quot;. $(LibraryToolsFolder)\CheckAssemblies.ps1 -BuildConfig $(Configuration) &quot;" Condition="'$(RunStaticAnalysis)' == 'true'" />
-
-    <!-- Cleanup extraneous files. -->
-    <Exec Command="$(PowerShellCoreCommandPrefix) &quot;. $(LibraryToolsFolder)\CleanupBuild.ps1 -BuildConfig $(Configuration)&quot;" />
-
-    <Exec ContinueOnError="false"
-          Command="$(PowerShellCoreCommandPrefix) &quot; . $(LibraryToolsFolder)\NewHelpIndex.ps1 -OutputFile $(PackageDirectory)\index.json -BuildConfig $(Configuration) &quot;" 
-          Condition="'$(CodeSign)' == 'true'"/>
-
-    <CallTarget Targets="BinScopeCheck" Condition="'$(OnPremiseBuild)'" />
-
-    <CallTarget Targets="RunPoliCheck" Condition="'$(OnPremiseBuild)'" />
-
-    <CallTarget Targets="CodeSignBinaries" Condition="'$(CodeSign)' == 'true'" />
-
-    <Exec ContinueOnError="false" Command="$(PowerShellCoreCommandPrefix) &quot;. $(LibraryToolsFolder)\NewOutputTypeIndex.ps1 -OutputFile $(PackageDirectory)\outputtypes.json -BuildConfig $(Configuration)&quot;" Condition="('$(Scope)' == 'All' or '$(Scope)' == 'Latest') and $(CodeSign) == 'true'" />
-
-    <Message Importance="high" Text="Running Static Analyser" />
-    <CallTarget targets="DependencyAnalysis" ContinueOnError="ErrorAndContinue" />
-    <!-- TODO: Implement CheckAssemblies.ps1 for Az. -->
-    <!-- <Exec Command="$(PowerShellCoreCommandPrefix) &quot;. $(LibraryToolsFolder)\CheckAssemblies.ps1 -BuildConfig $(Configuration) &quot;" /> -->
+    <PropertyGroup>
+      <RuntimeDllsIncludeList>Microsoft.Powershell.*.dll,System*.dll,Microsoft.VisualBasic.dll,Microsoft.CSharp.dll,Microsoft.CodeAnalysis.dll,Microsoft.CodeAnalysis.CSharp.dll</RuntimeDllsIncludeList>
+      <RuntimeDllsExcludeList>System.Security.Cryptography.ProtectedData.dll,System.Configuration.ConfigurationManager.dll,System.Runtime.CompilerServices.Unsafe.dll,System.IO.FileSystem.AccessControl.dll,System.Buffers.dll,System.Text.Encodings.Web.dll</RuntimeDllsExcludeList>
+    </PropertyGroup>
+    <Exec Command="$(PowerShellCoreCommandPrefix) &quot;Get-ChildItem -Path $(PackageDirectory)/$(Configuration) -Recurse -Include $(RuntimeDllsIncludeList) -Exclude $(RuntimeDllsExcludeList) | Remove-Item -Recurse -Force&quot;" />
```

Comment:
Isn't this redundant with CleanupBuild.ps1?

Reply:
As of the NetStandard switch, we have 2 cleanup processes. This process cleans up the framework files that are included. CleanupBuild.ps1, at this point, primarily removes duplicate shared DLLs from those published as part of Accounts. The reason it is 2 processes:
1. Removing the framework DLLs allows us to reduce the artifacts size for doing the ADO pipelines
2. If we did CleanupBuild.ps1 earlier, we cannot do Static Analysis on the modules, as they analyze each module independently. So, you cannot load the module if you're missing our DLLs, but you can load the module without the framework DLLs, since PowerShell itself has anything missing outside of the .NET Core SDK. The only reason the framework DLLs are here anyway is because of us not revising our build process to not use `CopyLocalLockFileAssemblies`.
3. At this point, CleanupBuild.ps1 only makes sense while publishing, so it is only ran at that point.